src/new.rs

```diff
@@ -1,3 +1,5 @@
+use crate::print_progress;
```

Comment:
This is not needed, the `print_progress` macro is only useful when waiting for something that takes time.

Reply:
Oh, I forgot to delete this line.